app/assets/javascripts/factories/service/service_data.js

```diff
@@ -0,0 +1,17 @@
+ManageIQ.angular.app.factory('serviceDataFactory', ['$http', function($http) {
```

Comment:
this file is never used and should probably go away.. (servies/service_data is used)